transitions/registration.js

```diff
@@ -272,5 +252,43 @@ module.exports = {
                 callback();
             }
         });
+    },
+    addPatient: function(options, callback) {
+        var doc = options.doc,
+            db = options.db,
+            audit = options.audit,
+            id = doc.patient_id,
+            patientNameField = (options.params && options.params.length && options.params[0]) || 'patient_name';
+
+        utils.getRegistrations({
+            db: db,
+            id: id
+        }, function(err, registrations) {
+            if (err) {
+                return callback(err);
+            }
+            if (registrations.length) {
+                // patient already registered, no action required
+                return callback();
+            }
+            db.medic.view('medic-client', 'people_by_phone', {
```

Comment:
hmmm. The point is to make it easier to spot and modify the strings you depend on, later. 
But! This probably is a java habit that makes less sense in javascript, since in javascript all names of everything are potentially typo-able, so strings don't really deserve special treatment. Interesting. Ok!